src/app/detail/ui/ActivityLog.tsx

```diff
@@ -1,30 +1,48 @@
-import { ActivityType } from '@/types/interfaces'
 import { Avatar, Box, Stack } from '@mui/material'
 import { BoldTypography, StyledTypography, VerticalLine } from '@/app/detail/ui/styledComponent'
 import { getTimeDifference } from '@/utils/getTimeDifference'
-import { mockActivitiesInterface } from '@/utils/mockData'
+import { LogResponse } from '@/app/api/activity-logs/schemas/LogResponseSchema'
+import { TaskAssignedResponse, TaskAssignedResponseSchema } from '@/app/api/activity-logs/schemas/TaskAssignedSchema'
+import { WorkflowStateUpdatedSchema } from '@/app/api/activity-logs/schemas/WorkflowStateUpdatedSchema'
 
 interface Prop {
-  log: mockActivitiesInterface
+  log: LogResponse
+}
+
+enum LogType {
+  TASK_CREATED = 'TASK_CREATED',
+  TASK_ASSIGNED = 'TASK_ASSIGNED',
+  WORKFLOW_STATE_UPDATED = 'WORKFLOW_STATE_UPDATED',
+}
```

Comment:
@arpandhakal @aatbip looks like we are not following same pattern for defining enum, I see different pattern in [this pr](https://github.com/copilot-platforms/tasks-app/blob/3176f52946816716d65cfbdb32012584be4333cf/src/app/api/core/types/tasks.ts)

Reply:
Nice catch @saugatmaharjan